subsys/net/lib/nrf_cloud/src/nrf_cloud_codec.c

```diff
@@ -1522,3 +1522,39 @@ int nrf_cloud_parse_cell_pos_response(const char *const buf,
 	cJSON_Delete(cell_pos_obj);
 	return ret;
 }
+
+bool nrf_cloud_detect_disconnection_request(const char *const buf)
+{
+	if (buf == NULL) {
+		return false;
+	}
+
+	/* The candidate buffer must be a null-terminated string less than
+	 * a certain length
+	 */
+	if (memchr(buf, '\0', NRF_CLOUD_JSON_MSG_MAX_LEN_DISCONNECT) == NULL) {
```

Comment:
Couldn't you reduce this to the size of the cloud DISCON message?  200 seems too large given that this is a function that is only for checking for that specific message.

Reply:
Ah, yes. The message we are looking for is typically 50 characters, not 150 as I thought. I will set the limit to 75.

Reply:
Done!